src/truffle-integration/projectsWatcher.js

```diff
@@ -1,3 +1,4 @@
+const { encodeEventSignature } = require("web3-eth-abi");
```

Comment:
All the changes in this file are already in develop (see https://github.com/trufflesuite/ganache/commit/4f8c534846057ddd0aa0973f19518f5c5d8323a0#diff-4e386a10b26b7224f48e8ee5a9652a3cR1-R129) and I can't seem to make them easily go away from this PR. You can ignore all changes in this file.